static/js/components/LearnerSearch.js

```diff
@@ -128,6 +129,16 @@ export default class LearnerSearch extends SearchkitComponent {
       {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
       LearnerResult
     );
+    this.forceSearch = true;
+  }
+
+  loaded = (): boolean => {
+    if (this.forceSearch) {
```

Comment:
ok, in that case I have two suggestions - the first is that the `forceSearch` variable be re-named to something more descriptive (perhaps `renderOnce`?), and the second is that the `loaded` method be renamed or separated from the behavior that sets the boolean value on `this`. `loaded` as a function name implies purity to me (just returning a value) and as it is written it has a side effect (it mutates the state that lives on the component instance, by setting `this.forceSearch`). 

it might be good also to leave a comment that explains what is going on and why the method is necessary, just so it's clear to others reading the code later.